python/tvm/target.py

```diff
@@ -521,6 +521,28 @@ def create(target_str):
     return _api_internal._TargetFromString(target_str)
 
 
+def get_mcpu(target):
```

Comment:
Is it possible to add this as a property under the Target class? Like `libs`

Reply:
You mean here 

https://github.com/dmlc/tvm/blob/02c1e11716a5afdfc0159a1b21e45a64f7875473/python/tvm/target.py#L84-L136


I was planning to, but then I saw that tvm.target.create is a separate function. I did not know why that was not a part of Target data structure. So, I decided to add as a helper. 

Do let me know if you have a better suggestion.

Reply:
LGTM